downloader/src/funTest/kotlin/MercurialTest.kt

```diff
@@ -44,6 +45,11 @@ class MercurialTest : StringSpec() {
     }
 
     init {
+        "Mercurial version is detected correctly" {
```

Comment:
Maybe rename to "Detected Mercurial version is not empty" as this is what is tested?

Reply:
Done.